apps/src/calc/blocks.js

```diff
@@ -49,35 +46,11 @@ exports.install = function(blockly, blockInstallOptions) {
 
 };
 
-function initFunctionalBlock(block, title, numArgs) {
-  block.setHSV(184, 1.00, 0.74);
-  block.setFunctional(true, {
-    headerHeight: 30,
-  });
-
-  var options = {
-    fixedSize: { height: 35 },
-    fontSize: 25 // in pixels
```

Comment:
Probably good to get this using the same code as the blockly-core version, but I'm not sure I'm convinced that they're exactly the same right now. For example, I dont think we get fontSize: 25 using the blockly-core version